rllib/algorithms/algorithm_config.py

```diff
@@ -765,6 +766,12 @@ def validate(self) -> None:
                 "`config.rollouts(enable_connectors=True)`."
             )
 
+        if bool(os.environ.get("RLLIB_ENABLE_RL_MODULE", False)):
```

Comment:
use command line arg

Reply:
Actually I had it like what you said and changed it to this way. It will be easier for us to flip it by default later on. Also adding that flag to every other unittest down the line will create a lot of duplicates and is not very clean for a temporary transition. This is a temporary solution until RLModule is proven out across the board and fully rolled out.